app/scripts/migrations/091.test.js

```diff
@@ -0,0 +1,149 @@
+import { migrate, version } from './091';
+
+jest.mock('uuid', () => {
+  const actual = jest.requireActual('uuid');
+
+  return {
+    ...actual,
+    v4: jest.fn(),
+  };
+});
+
+describe('migration #91', () => {
+  it('should update the version metadata', async () => {
+    const oldStorage = {
+      meta: {
+        version: 90,
+      },
+      data: {},
+    };
+
+    const newStorage = await migrate(oldStorage);
+    expect(newStorage.meta).toStrictEqual({
+      version,
+    });
+  });
+
+  it('should return state unaltered if there is no network controller state', async () => {
+    const oldData = {
+      other: 'data',
+    };
+    const oldStorage = {
+      meta: {
+        version: 90,
+      },
+      data: oldData,
+    };
+
+    const newStorage = await migrate(oldStorage);
```

Comment:
If we're comparing the new state to the old state later, we should clone it before passing it in:

```suggestion
    const newStorage = await migrate(cloneDeep(oldStorage));
```

Otherwise the test would fail to notice if the state was changed by mutation.

Reply:
The same applies for the other examples of this as well.

I think in practice this won't be a problem here because the data is cloned as part of our migration template, but it's still something that could be removed/missed in theory.

Reply:
done in 6cb2644fdb